### DISCUSSION TITLE
Feature ideas

### DISCUSSION THREAD

**Discussion by JorgenSolli** [MohmmedAshraf/laravel-translations#109] [Ideas]
Hey! Loving the new version. I've been using it for a bit, and I figured I'd share some thoughts.

## Sort translations by ID or created_at

After creating a new translation, it would be easier to quickly find it in your other languages.
 
## Change language from a specific translation

When viewing a specific translation from for example English, it would be nice to be able to change the current language to any of your other languages without having to go back to the dashboard and click the language, search for the phrase, and then click on it. Would save a lot of clicks.

## Chain-adding translations

I often add a lot of new translation keys, and come back after a bit to translate everything.
It would be nice to be able to go to the next untranslated phrase after saving one.

Something like "Save and go to next". You could save a lot of time by having a flow like that.

**Comment by MohmmedAshraf** [comment]
Hello @JorgenSolli, I appreciate your support definitely like the ideas and will begin implementing some of them soon
